Angular NgRx Starter fails CI

On the `main` branch, if I `cd` into `starters/angular-ngrx-scss` and run

```sh
npm i
```

...it fails with the following:

```sh
▶ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: angular-ngrx-scss@0.1.0
npm ERR! Found: zone.js@0.12.0
npm ERR! node_modules/zone.js
npm ERR!   zone.js@"0.12.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer zone.js@"~0.11.4" from @angular/core@13.3.12
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"13.3.12" from the root project
npm ERR!   peer @angular/core@"13.3.12" from @angular/animations@13.3.12
npm ERR!   node_modules/@angular/animations
npm ERR!     @angular/animations@"13.3.12" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

So any change that would cause the tests or build for `angular-ngrx-scss` to run are broken

--- a/starters/angular-ngrx-scss/package.json
+++ b/starters/angular-ngrx-scss/package.json
@@ -40,7 +40,7 @@
     "@ngrx/store-devtools": "13.2.0",
     "rxjs": "7.8.0",
     "tslib": "2.5.0",
-    "zone.js": "0.12.0"
+    "zone.js": "0.11.4"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "13.3.10",

